src/main/java/sx/blah/discord/handle/impl/obj/Guild.java

```diff
@@ -847,6 +845,26 @@ public IEmoji getEmojiByName(String name) {
 	}
 
 	@Override
+	public IEmoji createEmoji(String name, Image image, IRole... roles) {
+		if (name.length() < 2 || name.length() > 32 || DiscordUtils.EMOJI_CREATE_NAME.matcher(name).find())
+			throw new DiscordException("Emoji names must be at least 2 characters long and can only contain alphanumeric characters and underscores.");
```

Comment:
I would change this message so it's more in line to that of `IGuild#createChannel` or `IChannel#changeName`.